app/models/concerns/last_menstrual_period_measureable.rb

```diff
@@ -30,7 +30,11 @@ def _last_menstrual_period_on_date(date)
     return nil unless initial_call_date && last_menstrual_period_weeks
     weeks = 7 * (last_menstrual_period_weeks || 0)
     days = (last_menstrual_period_days || 0)
-    (date - initial_call_date) + weeks + days
+    if (date - initial_call_date) + weeks + days > 280
```

Comment:
Might want to just dry this out into another quick helper method since you repeat it.